http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -504,6 +518,7 @@ class GKEInterpreter[F[_]](
                 }
             } yield ()
         case AppType.Cromwell => persistentDiskQuery.updateLastUsedBy(diskId, app.id).transaction
+        case AppType.RStudio  => F.unit // TODO grab the pv and pvc
```

Comment:
Is there a follow-up ticket for this?

Reply:
Good catch, I do not believe that we need to support PD for RStudio here, but if we do it will be part of this PR so no need for a follow up ticket